src/tasks/WasmBuildTasks/UpdateChromeVersions.cs

```diff
@@ -285,16 +285,14 @@ private async Task<Stream> GetDownloadFileStreamAsync(string filename, string ur
                                                     .ConfigureAwait(false);
             if (response.StatusCode == HttpStatusCode.OK)
             {
-                string baseUrlForDownload = $"{s_snapshotBaseUrl}/{osPrefix}";
-                string snapshotUrl = $"{baseUrlForDownload}/{branchPosition}";
-                Log.LogMessage(MessageImportance.Low, $"Found {url}. Snapshots should be under ${snapshotUrl}");
-                return snapshotUrl;
+                Log.LogMessage(MessageImportance.Low, $"Found {url}");
```

Comment:
might be good to still log branchUrl here like how we logged snapshotUrl before. minor nit though